.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+Soundtrack of this PR: [link to song that really fits the mood of this PR]()
+
+### In this PR
+* < Additions, removals, fixes, features & functionality >
```

Comment:
I think it should remind people why additions/removals took place. "This is needed because..."

Reply:
Excellent, will update